guides/writing_a_training_loop_from_scratch.py

```diff
@@ -515,7 +515,7 @@ def train_step(real_images):
             print("adversarial loss at step %d: %.2f" % (step, g_loss))
 
             # Save one generated image
-            img = keras.utils.array_to_img(generated_images[0] * 255.0, scale=False)
+            img = tf.keras.utils.array_to_img(generated_images[0] * 255.0, scale=False)
```

Comment:
Keras is imported in this file so you don't need to change this.

Did you mean that the .md and .ipynb files need to be regenerated?

Reply:
yeah, you are right , I just did this to highlight this line as it is not showing in document.

Reply:
I'll update the website. Thanks for the notice!